modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -99,7 +105,7 @@ class SeqTranslate(site: Site) {
     val configs = sequenceConfig.getAllSteps.toList
 
     val steps = configs.zipWithIndex.traverseU {
-      case (c, i) => step(systems, settings)(obsId, i, c)
+      case (c, i) => step(systems, settings)(obsId, i, c, i==(configs.length-1))
```

Comment:
![good-let-the-hate-flow-through-you](https://cloud.githubusercontent.com/assets/3615303/23525325/d8bc05e8-ff6c-11e6-83c2-772d95f3b37f.jpg)